### ISSUE TITLE
[Settings]: At 225% Resize text, Browse and Add new Button text is not fully visible.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#11258]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insight for Windows

### Steps to reproduce

Prerequisites:
Open Settings app using 'Windows + i' Key.
Navigate to System and activate it.
Then Set Display Resolution to 1280 x 1024.
Then Navigate to Accessibility>Text Size.
Increase text size to 225% and observe the issue.

**Repro Steps:**
Open Windows Terminal.
Open Settings page using "Ctrl + ',' ".
Observe the button with same UI as Browse button is not visible properly.

**User Experience:**
Low Vision user will be impacted here as they will not be able to see button text as result not able use app properly.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/resize-text

**Attachments:**
[At 225% Resize text, Buttons text is not fully visible..zip](https://github.com/microsoft/terminal/files/7184124/At.225.Resize.text.Buttons.text.is.not.fully.visible.zip)

### Expected Behavior

At 225% Resize text, All Button text should be fully visible.

### Actual Behavior

At 225% Resize text, 'Browse' and 'Add new' Button text is not fully visible. Issue repro with all controls with similar name or UI.

**Comment by Priyanka-Chauhan123** [comment]
Closing the bug as UI has been changed.
